WordPress/gutenberg
# Try a masonry layout on the template pattern suggestion modal

## What?
Trying a 'masonry' layout of patterns in the suggestion modal when creating a new template, as an alternative approach to the current layout where all previews are the same size.

## Why?
* One-size-fits-all doesn't apply very well to patterns.
* We use a masonry layout in some other scenarios already.
* The masonry layout generally allows more patterns to be visible at once.

## How?
CSS

## Testing Instructions
* Create a template for which associated patterns exist
* Observe those patterns arranged in a 'masonry' layout


## Before

https://user-images.githubusercontent.com/846565/233380138-dcfa42b7-d2ee-46e1-949d-ad45b4d4de85.mp4



## After

https://user-images.githubusercontent.com/846565/233380780-fdf2d8bf-1d9c-43a8-ad14-375f178a747a.mp4
